draft-ietf-rats-architecture.md

```diff
@@ -1330,7 +1326,7 @@ boot, or immutable hardware/ROM.
 It is also important that the appraisal policy was itself obtained securely.
 If an attacker can configure appraisal policies for a Relying Party or for a Verifier, then integrity of the process is compromised.
 
-The security protecting conveyed information may be applied at different layers, whether by a conveyance protocol, or an information encoding format. This architecture expects attestation messages (i.e., Evidence, Attestation Results, Endorsements, Reference Values, and Policies) are end-to-end protected based on the role interaction context.
+Security protected conveyed information in RATS may be applied at different layers, whether by a conveyance protocol, or an information encoding format. This architecture expects conceptual messages (see {{messages}}) to be end-to-end protected based on the role interaction context.
```

Comment:
protecting vs. protected. As Giri just quoted this exact snippet and seemed to have no problems with it at all. Maybe I simply did not get the original meaning? Nevertheless, as it confuses me, maybe it also confuses other non-native speakers?

Reply:
I can parse the original (red) start of this paragraph but I cannot parse the green change.